### DISCUSSION TITLE
open config via cli when opening gui

### DISCUSSION THREAD

**Discussion by Some1OnLine** [wiresock/WireSockUI#35] [Ideas]
if not already available (and I simply missed it), it would be nice to be able to open the gui from cli to include a specific config.

**Comment by wiresock** [comment]
Hello,

Thank you for your suggestion about the ability to open the GUI with a specific configuration from the command line interface (CLI). To ensure we fully understand and address your request, could you please provide more details?

Are you looking for a feature that allows importing a specific configuration file into the GUI directly via the CLI? Alternatively, are you suggesting an enhancement where the CLI can initiate the GUI with a pre-selected configuration already in place? For the latter, you might find the WireSock VPN Client CLI useful, as it already supports similar functionality.

Further details about your specific use case or the workflow you have in mind would be incredibly valuable. This information will help us better assess your suggestion and determine how it might fit into our development roadmap.

Looking forward to hearing more from you.

Best regards,
Vadim
